src/components/modal/index.tsx

```diff
@@ -46,6 +48,10 @@ export interface ModalProps extends RNModalProps {
    * Should add a GestureHandlerRootView (Android only)
    */
   useGestureHandlerRootView?: boolean;
+  /**
+   * Send in order to add a KeyboardAvoidingView (iOS only)
+   */
+  keyboardAvoidingViewProps?: KeyboardAvoidingViewProps;
```

Comment:
Should we split it into two props? `useKeyboardAvoidingView` and `keyboardAvoidingViewProps`
What if the user doesn't need/want to pass any props to it, but just enable it?

Reply:
I was not sure about this choice but technically the user can send an empty object (which will work), WDYT?

Reply:
I'm not a fan, it's not clear enough and it force the user to pass something that isn't trivial
I think in this case I'd go with two props.